README.md

```diff
@@ -505,6 +505,8 @@ dataFrameName: test
     alignToPreviousSchema: true
     # Remove completely null columns
     removeNullColumns: true
+    # Delete inflight and compaction requested of unfinished commit
+    deletePendingCompactions: true
```

Comment:
I think it should be part of the job config right?

Reply:
I think it should be in Hudi output options. No?

Reply:
Well yeah but just in the job level, not metric level, there are 2 output configs

Reply:
Also add it to the example yaml